linux_os/guide/system/accounts/accounts-pam/password_quality/password_quality_pwquality/accounts_password_pam_retry/tests/argument_missing.fail.sh

```diff
@@ -1,13 +1,6 @@
 #!/bin/bash
-# platform = Red Hat Enterprise Linux 7,Red Hat Virtualization 4,multi_platform_fedora,multi_platform_ol,multi_platform_ubuntu
+# packages = pam
```

Comment:
Is there any case where `pam` package is not installed (or even possible to be removed) in any of the distros mentioned in `prodtype`? Including a test for his package in the test scenarios scripts is not a problem, but doesn't seems necessary. Maybe we can save some seconds during the tests.